crates/criticalup-cli/src/commands/install.rs

```diff
@@ -90,7 +91,9 @@ async fn install_product_afresh(
     let release_manifest_from_server = client
         .get_product_release_manifest(product_name, product.release())
         .await?;
-    let verified_release_manifest = release_manifest_from_server.signed.into_verified(&keys)?;
+    let verified_release_manifest = release_manifest_from_server
+        .signed
+        .into_verified(&keys, revocation_info)?;
```

Comment:
This is effectively this?
```suggestion
        .into_verified(&keys, keys.revocation_info())?;
```

Reply:
Same as above.